src/test/kotlin/no/nav/familie/ef/sak/fagsak/FagsakPersonServiceTest.kt

```diff
@@ -82,4 +114,161 @@ internal class FagsakPersonServiceTest : OppslagSpringRunnerTest() {
             .isInstanceOf(Feil::class.java)
             .hasMessageContaining("Finner ikke personidenter")
     }
+
+    @Nested
+    inner class AktiveringAvMikrofrontend {
+        @Test
+        fun `skal finne fagsakperson gitt fagsakId`() {
+            testoppsettService.opprettPerson("9")
+            val fp = testoppsettService.opprettPerson("7")
+            val fagsakOS = testoppsettService.lagreFagsak(fagsak(fagsakPersonId = fp.id, stønadstype = OVERGANGSSTØNAD))
+            val fagsakBT = testoppsettService.lagreFagsak(fagsak(fagsakPersonId = fp.id, stønadstype = BARNETILSYN))
+            val fpOS = fagsakPersonService.finnFagsakPersonForFagsakId(fagsakOS.id)
+            val fpBT = fagsakPersonService.finnFagsakPersonForFagsakId(fagsakBT.id)
+
+            assertThat(fpOS).isEqualTo(fp)
+            assertThat(fpBT).isEqualTo(fp)
+            assertThrows<Exception> { fagsakPersonService.finnFagsakPersonForFagsakId(UUID.randomUUID()) }
+        }
+    }
+
+    @Nested
+    inner class DeaktiveringAvMikrofrontend {
+        val mindreEnnEnMånedSiden = LocalDateTime.now().minusMonths(1).plusDays(1)
+        val merEnnEnMånedSiden = LocalDateTime.now().minusMonths(1).minusDays(1)
+        val merEnnSeksMånederSiden = LocalDateTime.now().minusMonths(6).minusDays(1)
+        val mindreEnnSeksMånederSiden = LocalDateTime.now().minusMonths(6).plusDays(1)
+        val mindreEnnFireÅrSiden = LocalDateTime.now().minusYears(4).plusDays(1)
+        val merEnnFireÅrSiden = LocalDateTime.now().minusYears(4).minusDays(1)
+
+        @Test
+        fun `skal ikke finne noen å deaktivere hvis ingen er aktive`() {
+            testoppsettService.opprettPerson("12345")
+            val personIderForDeaktivering = fagsakPersonService.finnFagsakpersonIderKlarForDeaktiveringAvMikrofrontend()
+            assertThat(personIderForDeaktivering).isEmpty()
+        }
+
+        @Test
+        fun `skal deaktivere mikrofrontend hvis fagsakperson ikke har behandlinger og er opprettet for mer enn 1 mnd siden`() {
+            val forNyFagsakPerson = opprettFagsakPerson("12345", opprettetTid = mindreEnnEnMånedSiden)
```

Comment:
`fornyFagsakPerson` og `fagsakPersonAldriAktivert` blir aldri brukt her. Kanskje lage en test på at man IKKE deaktiverer minside for en person dersom han oppfyller kriteriene for deaktivering men har `harAktivertMikrofrontend` satt til false?